cli/ballerina-cliutils/pom.xml

```diff
@@ -0,0 +1,186 @@
+<!--
```

Comment:
shall we rename cliutils to cli-utils everywhere.

Reply:
Fixed with 9fcdece5175329f9bb65d427cb38db2c9f44d701